app/views/collection/edit_look.html.slim

```diff
@@ -32,27 +32,30 @@
             =f.label :hide_completed
               h3 =t('.hide_completed')
               p =t('.hide_completed_description')
-        tr 
+        tr
           td(colspan="2")
             =f.check_box :messageboards_enabled
             =f.label :messageboards_enabled
               h3 =t('.enable_forums')
               p =t('.forums_description')
         -if current_user.account_type != "Individual Researcher"
-          tr 
+          tr
             td(colspan="2")
               =f.check_box :supports_document_sets
               =f.label :supports_document_sets
                 h3.settings-label =t('.enable_document_sets')
                 =link_to t('.edit_document_sets'), document_sets_path(collection_id: @collection), disabled: !@collection.supports_document_sets?, class: 'button inline-btn exits', id: 'document-sets-edit', 'aria-label' => t('.edit_document_sets')
                 p =t('.document_sets_description')
-        tr 
+        tr
           td(colspan="2")
-            h3 =t('.metadata')
+            =f.label :metadata, style: 'margin-bottom: 5px;'
+              h3.settings-label =t('.metadata')
```

Comment:
We need to conditionalize this so that we don't display it if it isn't applicable.  Here are the relevant scenarios:
* _most common_ All works in the collection were uploaded as PDFs or ZIP files and do not have corresponding IIIF manifests, so we should not display any refresh button.
* _common_ All works in the collection were imported from IIIF manifests.  In this case we want to display the button.
* _rare_ Some works in the collection were imported from IIIF; others were uploaded.  In this case we might want to display the button, but it's OK if we don't.


You can tell that a work was imported via IIIF if it has a `ScManifest` object, so logic like `@collection.works.detect? {|w| w.sc_manifest.present? }` may be enough.